.github/workflows/manual.deploy.nightly.devnet.yml

```diff
@@ -3,9 +3,9 @@
 
 name: Nightly DevNet Workflow (Manual)
 on:  # yamllint disable-line rule:truthy
-  # push:
-  #   branches:
-  #     - jesse/fix-nightly-build
+  push:
+    branches:
+      - jesse/fix-*
```

Comment:
Should this be removed?

Reply:
i can remove this anytime. i feel like i keep adding this back in whenever i fix the nightly so i just kept it in for now. i'll remove it soon.